examples/first-frame/main.cpp

```diff
@@ -41,9 +42,35 @@
 #endif
 #include <ios>
 #include <iostream>
+#include <map>
 
 using namespace aditof;
 
+static const char Help_Menu[] =
+    R"(First-frame usage:
+    first-frame CONFIG
+    first-frame (-h | --help)
+    first-frame [-ip | --ip <ip>] [-m | --m <mode>] CONFIG
+
+    Arguments:
+      CONFIG            Input config_default.json file (which has *.ccb and *.cfg)
+
+    Options:
+      -h --help          Show this screen.
+      -m --m <mode>      Mode to capture data in. [default: 0]
+
+    NOTE: -m | --m argument supports both index and string (0/sr-native)
+
+    Valid mode (-m | --m) options are:
+        0: short-range native
+        1: long-range native
+        2: short-range Qnative
+        3: long-range Qnative
+        4: pcm-native
+        5: long-range mixed
+        6: short-range mixed
+    )";
```

Comment:
Might want to remove any extra space  before `)`, or perhas adding an extra new line. This way the new command line wont be disalligned

Reply:
Solved